File: Backend/src/main/java/ssafy/d210/backend/dto/response/user/LoginResponse.java
package ssafy.d210.backend.dto.response.user;

import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class LoginResponse {
    // 이 부분이 성윤님이 쓴 부분
    private String email;
    private String password;

    // 이 부분이 한나가 쓴 부분
    private long userId;
    private String nickname;
    private int certificateCount;
    private String wallet;
}

File: Android/app/src/main/java/com/example/second_project/data/model/dto/response/LoginResponse.kt
package com.example.second_project.data.model.dto.response

data class LoginResponse(
    val code: Int,
    val `data`: Data,
    val status: String,
    val timeStamp: String
)

data class Data(
    val certificateCount: Int,
    val nickname: String,
    val userId: Int,
    val wallet: String,
    val name : String
)